sources/packages/backend/apps/workers/src/controllers/disbursement/_tests_/e2e/disbursement.controller.saveDisbursementSchedules.e2e-spec.ts

```diff
@@ -344,6 +344,158 @@ describe("DisbursementController(e2e)-saveDisbursementSchedules", () => {
     assertOveraward(overawards, "CSLP", 300);
   });
 
+  it("Should assert 'hasEstimatedAward' flag as false when disbursement value amounts are zero.", async () => {
+    // Arrange
+    const savedUser = await db.user.save(createFakeUser());
+    const savedStudent = await db.student.save(createFakeStudent(savedUser));
+    const fakeApplication = createFakeApplication(
+      { student: savedStudent },
+      { initialValue: { applicationStatus: ApplicationStatus.InProgress } },
+    );
+    fakeApplication.applicationNumber = "OA_TEST003";
+    const savedApplication = await db.application.save(fakeApplication);
+    // Original assessment.
+    const fakeOriginalAssessment = createFakeStudentAssessment({
+      auditUser: savedUser,
+    });
+    fakeOriginalAssessment.application = savedApplication;
+    const savedOriginalAssessment = await db.studentAssessment.save(
+      fakeOriginalAssessment,
+    );
+    const saveDisbursementSchedulesPayload =
+      createFakeSaveDisbursementSchedulesPayload({
+        assessmentId: savedOriginalAssessment.id,
+        createSecondDisbursement: true,
+        firstDisbursementAwards: [
+          {
+            valueCode: "CSLF",
+            valueType: DisbursementValueType.CanadaLoan,
+            valueAmount: 0,
+          },
+          {
+            valueCode: "BCSL",
+            valueType: DisbursementValueType.BCLoan,
+            valueAmount: 0,
+          },
+        ],
+        secondDisbursementAwards: [
+          {
+            valueCode: "CSLF",
+            valueType: DisbursementValueType.CanadaLoan,
+            valueAmount: 0,
+          },
+          {
+            valueCode: "BCSL",
+            valueType: DisbursementValueType.BCLoan,
+            valueAmount: 0,
+          },
+        ],
+      });
+    const saveResult = await disbursementController.saveDisbursementSchedules(
```

Comment:
Minor. Add the //Act comment before line 394. 

![image](https://github.com/user-attachments/assets/2e4a5374-663e-4dfb-aa59-968acc3fdf7d)